run/helm/snap/charts/grafana/templates/job_cluster.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
```

Comment:
Okay, that clarifies a lot for me. I will look into using Wizzy, but I would still need to go through the pros and cons of using a script inside the container.

As of right now this implementation seems fine.